2d_classification/monai_101.ipynb

```diff
@@ -191,18 +216,29 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Prepare datasets using MONAI Apps\n",
+    "## Prepare Dataset Using MONAI Apps\n",
+    "\n",
+    "We'll use the `MedNISTDataset` from MONAI Apps to automatically download and set up our dataset. This convenience class will:\n",
+    "- Download the dataset to your specified directory\n",
+    "- Apply the preprocessing transforms we defined above\n",
+    "- Split the data into training, validation, and test sets\n",
     "\n",
-    "We use `MedNISTDataset` in MONAI Apps to download a dataset to the specified directory and perform the pre-processing steps in the `monai.transforms` compose.\n",
+    "### About the MedNIST Dataset\n",
     "\n",
-    "The MedNIST dataset was gathered from several sets from [TCIA](https://wiki.cancerimagingarchive.net/display/Public/Data+Usage+Policies+and+Restrictions),\n",
-    "[the RSNA Bone Age Challenge](http://rsnachallenges.cloudapp.net/competitions/4),\n",
-    "and [the NIH Chest X-ray dataset](https://cloud.google.com/healthcare/docs/resources/public-datasets/nih-chest).\n",
+    "The MedNIST dataset is a collection of medical images from multiple sources:\n",
+    "- [TCIA](https://wiki.cancerimagingarchive.net/display/Public/Data+Usage+Policies+and+Restrictions) (The Cancer Imaging Archive)\n",
+    "- [RSNA Bone Age Challenge](http://rsnachallenges.cloudapp.net/competitions/4)\n",
+    "- [NIH Chest X-ray Dataset](https://cloud.google.com/healthcare/docs/resources/public-datasets/nih-chest)\n",
```

Comment:
```suggestion
```
These links don't really make sense, it's best to just leave the link to the Mayo page as in the original, further details can be found there.